### ISSUE TITLE
scmMutationCoverage fails when only interfaces are present in changeset

### ISSUE THREAD

**Issue by Kwaq** [hcoles/pitest#306]
When changed class is interface (with no default methods) and there are no other concrete classes scmMutationCoverage fails with error below.

Same will happen if we explicitly provide such interface as target class. 

org.pitest.help.PitHelpError: No classes found with line number debug information. All classes should be compiled with source and line number debug information.
See http://pitest.org for more details.

``` java
    at org.pitest.mutationtest.verify.DefaultBuildVerifier.checkAtLeastOneClassHasLineNumbers(DefaultBuildVerifier.java:44)
    at org.pitest.mutationtest.verify.DefaultBuildVerifier.verify(DefaultBuildVerifier.java:35)
    at org.pitest.mutationtest.tooling.MutationCoverage.verifyBuildSuitableForMutationTesting(MutationCoverage.java:221)
    at org.pitest.mutationtest.tooling.MutationCoverage.runReport(MutationCoverage.java:108)
```

**Comment by KyleRogers** [comment]
I think this is a valid error message when explicitly providing such interface as a target class. At least there is nothing to mutate.

But I agree that this verification should be lowered when using scmMutationCoverage goal. Maybe there can be a pre-check that skips mutation coverage execution if changes will be found in interfaces (without default and static methods) only.

**Comment by hcoles** [comment]
Released in 1.2.0
